app-modules/contact/src/Models/OrganizationIndustry.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Aiding App™ is licensed under the Elastic License 2.0. For more details,
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AidingApp\Contact\Models;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Concerns\HasUuids;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class OrganizationIndustry extends Model
+{
+    use HasFactory;
+    use HasUuids;
+    use SoftDeletes;
+
+    protected $fillable = [
+        'name',
+        'is_default',
+    ];
+
+    public function organisations(): HasMany
+    {
+        return $this->hasMany(Organization::class, 'industry_id');
+    }
+
+    public function createdBy(): BelongsTo
```

Comment:
@dgoetzit, @danharrin, and @mxm1070 can you help me remember if `createdBy` is needed on any of theses Models?

I think we only added it to some very specific Models at one point.

Reply:
@Orrison I don't see it on the ticket so we can probably remove it from these models.

Reply:
Hi @Orrison @mxm1070 
we cannot remove this because we are using that in observer to associate user in created_by_id field. see below code.
($user instanceof User) ? $organization->createdBy()->associate($user) : '';

Reply:
@ketan-canyon Remove that too.

Reply:
@mxm1070 
Removed created_by_id fields  from  table for all three new models